packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -133,18 +133,24 @@ export class HbarLimitService implements IHbarLimitService {
     );
 
     // Reset the rate limiter at the start of the next day
-    const now = Date.now();
-    const tomorrow = new Date(now + HbarLimitService.ONE_DAY_IN_MILLIS);
-    this.reset = new Date(tomorrow.setHours(0, 0, 0, 0));
+    this.reset = this.getResetTimestamp();
   }
 
   /**
    * Resets the {@link HbarSpendingPlan#spentToday} field for all existing plans.
+   * @param {string} [requestId] - An optional unique request ID for tracking the request.
    * @returns {Promise<void>} - A promise that resolves when the operation is complete.
    */
-  async resetLimiter(): Promise<void> {
-    // TODO: Implement this with https://github.com/hashgraph/hedera-json-rpc-relay/issues/2868
-    throw new Error('Not implemented');
+  async resetLimiter(requestId?: string): Promise<void> {
```

Comment:
In the context of recent changes, shouldnt requestId be required or should I change it in my PR?

Reply:
Let's do this as part of https://github.com/hashgraph/hedera-json-rpc-relay/pull/2939, which would have to change this method to use a `RequestDetails` object anyway.